docs/troubleshooting.rst

```diff
@@ -193,3 +193,31 @@ Now build again::
     tutor images build
 
 All build commands should now make use of the newly configured builder. To later revert to the default builder, run ``docker buildx use default``. 
+
+Can't override styles using Indigo Theme for MFEs
+-------------------------------------------------
+
+The indigo theme can’t override styles for MFEs directly. It overrides the styles for edx-platform. In case of MFEs, `@edx/brand <https://github.com/openedx/brand-openedx>`_ is used to override the styles. Then, customize the ``@edx/brand`` package to your preferences and include this customized package in `tutor-indigo` plugin. In this way, styles can be overidden::
+
+
+    hooks.Filters.ENV_PATCHES.add_item((
+                "mfe-dockerfile-post-npm-install",
+                """
+    RUN npm install '@edx/brand@npm:custom-brand-package'
+    RUN npm install '@edx/brand@git+https://github.com/username/brand-openedx.git#custom-branch'
+    """,
+            ))
+
+
+NPM Dependency Conflict When overriding ``@edx/frontend-component-header`` or ``@edx/frontend-component-footer``
+----------------------------------------------------------------------------------------------------------------
+
+When there is a need to customize the ``@edx/frontend-component-header`` or ``@edx/frontend-component-footer`` component, two things we have to care of to avoid dependency conflict error during ``npm`` installation. 
+
+1. Identify your openedx version, for example ``quince``.
+2. Navigate to `learning <https://github.com/openedx/frontend-app-learning>`_ and `learner-dashboard <https://github.com/openedx/frontend-app-learner-dashboard>`_ MFEs repositories and checkout to branch ``quince``. Inspect which `header` and `footer` versions are installed from `package.json`. You can explore the `header` and `footer` versions for additional MFEs to ensure that all MFEs support the same `major` versions of `header` and `footer`. 
```

Comment:
Why are "header" and "footer" always surrounded by backticks? I don't think we need those, right?